app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
+import datetime
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor should be vaccinated!")
+        elif visitor["vaccine"]["expiration_date"] \
+                < datetime.date.today():
+            raise OutdatedVaccineError(
+                "Visitor's vaccine should not be outdated!")
+        elif visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("Visitor should wear a mask!")
+        else:
+            return f"Welcome to {self.name}"

```

Comment:
u can use just if here + unnecessary else at the end of method, u can leave just return